sdk/attestation/attestation/swagger/readme.md

```diff
@@ -1,21 +1,46 @@
-# Azure Cognitive Search TypeScript Protocol Layer
+# Azure Attestation Service client library for TypeScript
 
-> see https://aka.ms/autorest
+## Setup
 
-## Configuration
+```ps
+npm install -g autorest
+```
+
+## Generation
+
+```ps
+cd <swagger-folder>
+autorest README.md
+```
+
+### Code generation settings
 
 ```yaml
 package-name: "@azure/attestation"
-package-version: 1.0.0-beta.2
```

Comment:
keeping this flag will fix the bad version in the generated context class.

Reply:
I'm not opposed to that, but what keeps the various versions in sync? There are at least three of them that need to be synced...

Reply:
There is a bot that bumps the versions in package.json, src/generated/clientContext.ts, and src/constants.ts after each release. Once we migrate to core v2, we will no longer need to have the version in clientContext.ts.

Reply:
QQ: if the bot doesn't also update swagger/readme.md, what prevents a re-run of swagger generation from blowing this up?

Reply:
nothing other than human review of the PR and noticing the diff. I would not worry too much about this because it is going to go away once we move to core v2.